### ISSUE TITLE
Create overview page for different finders

### ISSUE THREAD

**Issue by bastiRe** [codeformunich/FinderApp#32]
Wir haben uns geeinigt eine eigene Übersichtsseite für die verschiedenen Versionen der FinderApp zu machen, die als eigentständiges Mini-Projekt gestartet wird.

**Comment by saerdnaer** [comment]
Ich hab jetzt mal http://codeformunich.github.io angelegt und mit einem Standard-Template vom Github Site-Generator befüllt. Der Inhalt kann unter https://github.com/codeformunich/codeformunich.github.io geändert werden. 
Die Fragen die jetzt noch beantwortet werden müssen:
* Wie soll diese Liste genau aussehen? Einfach die Icons untereinander mit einem Namen daneben?
* Wie wird diese Liste aus einer konkreten App heraus verlinkt? Verlinkte das Icon rechts oben auf http://codeformunich.github.io inzwischen aber direkt auf http://codeformunich.github.io/FinderApp . Sollen wir das wieder ändern oder ist der weg eh nicht wirklich offensichtlich?

**Comment by MarHai** [comment]
Fürs Erste fände ich eine simple Liste mit Icons ausreichend. In den Apps könnte man dann auf der About-Seite auf diese Liste verweisen oder einen zusätzlichen Menüpunkt einführen.

**Comment by bastiRe** [comment]
Ich stimme Mario zu. Einfach ne Liste oder Kacheln mit den verschiedenen Findern. Das Icon in der App würde ich weiterhin auf den Startpunkt der App verweisen lassen. Sobald die Übersichtsseite fertig ist können wir dann in der App einfach einen Menüpunkt einführen der auf die Übersichtsseite verlinkt.
